script/generate_build_info.py

```diff
@@ -131,9 +166,22 @@ def get_files_list(dirname:str, recursive:bool=True):
         if 'version' not in docker_details:
             docker_details ['version'] = '*'
         dockerfile_versions.append(docker_details)
-
-
+    elif file.endswith(".json"):
+
+        new_key_value = {"maintainer":"{}".format(get_maintainer_for_package(dir_name))}
```

Comment:
As the maintainer can be either from dockerfile or build-script, this variable should be based on the conditional value setting. Which is defined between line 179 - 189.

i.e change the line to `new_key_value = {"maintainer": maintainer}`